sdk/keyvault/keyvault-keys/src/tracing.ts

```diff
@@ -16,6 +16,6 @@ export const createSpan: <T extends {
   span: Span;
   updatedOptions: T;
 } = createSpanFunction({
-  packagePrefix: "KeyClient",
+  packagePrefix: "KeysClient",
```

Comment:
isn't it called KeyClient? https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/keyvault/keyvault-keys/src/index.ts#L189